### DISCUSSION TITLE
Composer patch versions should be marked as stable

### DISCUSSION THREAD

**Discussion by lukaskolafa** [renovatebot/renovate#23836] [Suggest an Idea]
### Tell us more.

Hello team,

I am referring to this issue: https://github.com/renovatebot/renovate/issues/8849

Magento composer patch versions should be considered as 'stable'. For example, `2.4.6-p2` is a stable version. Here is referenced the composer code that is relevant to the stability decision:

https://github.com/composer/semver/blob/fa1ec24f0ab1efe642671ec15c51a3ab879f59bf/src/VersionParser.php#L35

I created a fix for the issue and would like to propose a pull request. I am currently not allowed to create a branch. Could you please provide me a hint on how to proceed now?

thank you,
best regards
Lukas

**Comment by rarkins** [comment]
https://github.com/renovatebot/renovate/blob/main/docs/development/local-development.md#fork-and-clone

**Comment by lukaskolafa** [comment]
thanks, I was able to create a pull request, please contact me if there are some concerns or other improvements necessary: https://github.com/renovatebot/renovate/pull/23842
